src/locales/pt/translation.json

```diff
@@ -109,6 +109,8 @@
     "downloads_path": "Diretório dos downloads",
     "change": "Mudar",
     "notifications": "Notificações",
+    "menu_options": "Opções",
```

Comment:
Please add a more descriptive text about the section other than "Options"
Something like "Download options" for this specific scenario